clients/python/.gitignore

```diff
@@ -5,3 +5,4 @@
 /.nox/
 /.python-version
 __pycache__/
+.port-forwards.pid
```

Comment:
We don't want to accidentally commit the port-forward file if something goes wrong